library/swift/EngineBuilder.swift

```diff
@@ -169,6 +171,28 @@ public final class EngineBuilder: NSObject {
     return self
   }
 
+  /// Set a closure to be called when the engine's logger logs.
+  ///
+  /// - parameter closure: The closure to be called.
+  ///
+  /// - returns: This builder.
+  @discardableResult
+  public func setonEngineLog(closure: @escaping (String) -> Void) -> EngineBuilder {
+    self.onEngineLog = closure
+    return self
+  }
+
+  /// Set a closure to be called when the engine's logger flushes.
```

Comment:
What's the purpose of this callback if we're (presumably) logging each statement immediately?

Reply:
I tend to agree with @goaway here unless there is additional context as to why we would want to observe this specific behavior

Reply:
Discussed in https://github.com/envoyproxy/envoy-mobile/pull/1355. Not going to expose flushing.